arrows/pytorch/SRNN_matching.py

```diff
@@ -25,9 +25,7 @@ def __init__(self, track_set, track_state_list, track_search_threshold, rnnType)
     def _make_dataset(self):
         _data_list = []
 
-        for t in range(self._tracks_num):
-            cur_track = self._track_set[t]
-
+        for t, cur_track in enumerate(self._track_set):
```

Comment:
@neal-siekierski Good point. Since it was my first python coding work, I was still in the mindset of C++.